new/LIT7221QeneBeelomu.xml

```diff
@@ -0,0 +1,73 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7221QeneBeelomu" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ብዕሎሙ፡ውስተ፡ ሰማይ፡ ለነዳያነ፡ ምድር፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Bǝʿǝlomu wǝsta samāy la-nadāyāna mǝdr (Qǝne possibly of wāzemā-type)</title>
```

Comment:
Bǝʿǝlomu - I would transcribe Bǝʿlomu because this means "their wealth". The passage that makes a bit more sense for the title is then: ብዕሎሙ፡ ውስተ፡ ሰማይ፡ ለነዳያነ፡ ምድር፡ ሐዋርያት፡ "The wealth of the poor of the earth, the Apostles, is in the heaven ..." (I read it as:  ብዕሎሙ፡ ለነዳያነ፡ ምድር፡ ሐዋርያት፡ ውስተ፡ ሰማይ፡  (ውእቱ፡))

Reply:
OK.